integrations/openai/src/databricks_openai/vector_search_retriever_tool.py

```diff
@@ -97,7 +97,7 @@ def _validate_tool_inputs(self):
             raise ValueError(
                 f"Index name {self.index_name} is not in the expected format 'catalog.schema.index'."
             )
-        self._index = VectorSearchClient().get_index(index_name=self.index_name)
+        self._index = VectorSearchClient(disable_notice=True).get_index(index_name=self.index_name)
```

Comment:
Interesting that we are not allowing users pass `client_args`. But lets not block this PR for that. We can create a ticket to track this later on.